bin/node-sass

```diff
@@ -204,6 +246,15 @@ function run(options, emitter) {
     options.includePath = [options.includePath];
   }
 
+  if(isDirectory(options.src)) {
```

Comment:
Since below you are just relying on `options.directory` being set, seems like you could just do that here to.